tslint/tslint_annotations.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+const { getAnnotationLevel } = require('../annotations_levels')
+
+/**
+ * Translates the tslint severity from "ERROR" | "WARNING" | "OFF"
+ * to the "HIGH", "MEDIUM" and "LOW"
+ * @param {String} severity issues severity given by tslint
+ */
+function translateSeverity (severity) {
+  let result = ''
+  switch (severity) {
+    case 'ERROR': result = 'HIGH'; break
+    case 'WARNING': result = 'MEDIUM'; break
+    case 'OFF': result = 'LOW'
+  }
+  return result
+}
+
+/**
+ * Create TSLint Annotations
+ * @param {*} issue
+ * @returns {Object} see https://developer.github.com/v3/checks/runs/#annotations-object-1
+ */
+function getAnnotation (issue) {
+  const path = issue.name
+  // I add 1 to startLine and endline because they are off by 1 as described here:
+  // https://github.com/palantir/tslint/issues/4528
```

Comment:
I also generally don't add comments referring to bugs since that is what git log and git blame is for.